README.md

```diff
@@ -66,6 +66,8 @@ $ make install
 
 When crosstool-ng's maintainers tag a release after 1.24.0 readers can downgrade to that for stability.
 
+**Page 43**: http://www.sqlite.org/2020/sqlite-autoconf-3330000.tar.gz is no longer available. Replace it with an up-to-date source code URL from [SQLite Download Page](https://www.sqlite.org/download.html) and adjust the subsequent `tar` and `cd` commands' arguments.
```

Comment:
I was actually able to download `sqlite-autoconf-3330000.tar.gz` using that link today but I see your point. Better not to hardcode package URLs in the book.

Reply:
Oh my, you're right! Turns out I didn't copy the full URL from the book, got 404, and went looking for a recent version right away thinking the old one is no longer available.